src/lib/components/Modal/ModalFooter.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withGlobalProps } from '../../provider';
+import {
+  Toolbar,
+  ToolbarItem,
+} from '../Toolbar';
+import styles from './ModalFooter.scss';
+
+export const ModalFooter = ({
+  children,
+  id,
+  justify,
+}) => (
+  <div
+    className={styles.root}
+    id={id}
+  >
+    <Toolbar
```

Comment:
I think I would not include `Toolbar`, I would just render the `children` directly.

```jsx
<Toolbar
  dense
  justify={justify}
>
  {children}
</Toolbar>
```

I can envision usages where the footer might prefer other ways of organizing the content. For example the user might prefer to use some custom layout component

While I agree this would be nice for our use cases I think it would be too opinionated for RUI.

The same might go for the `justify` prop.